Marketplace Wishlist & Marketplace Promotions and Discounts (#178)
spryker/spryker.github.io: docs/marketplace/user/features/202106.0/marketplace-wishlist-feature-overview.md

---
title: Marketplace Wishlist feature overview

> description: This document contains concept information for the Marketplace Wishlist feature in the Spryker Cloud Commerce OS.

Reviewer: ```suggestion
description: This document contains concept information for the Marketplace Wishlist feature.
```

template: concept-topic-template
---

Wishlists are collections of products that a customer saves for further reference. 

> Marketplace Wishlist feature allows customers to add product offers and merchant products to a wishlist in the Marketplace Storefront. This way customers save time without the need of browsing favorite items all over the marketplace shop.

Reviewer: ```suggestion
The *Marketplace Wishlist* feature allows customers to add product offers and merchant products to a wishlist in the Marketplace Storefront. This way, customers save time without having to browse favorite items all over the marketplace shop.
```

![](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+Wishlist/add-products-and-offers-to-wishlist.gif)

Merchant information is displayed in the *Sold by* field so that customer can always know which merchant is the owner of the product. 

> Once the customer is committed to a purchase of a product, they can add it to cart directly from the wishlist.

Reviewer: ```suggestion
Once the customer decides to buy a product, they can add it to cart directly from the wishlist.
```

![](https://spryker.s3.eu-central-1.amazonaws.com/docs/Marketplace/user+guides/Features/Marketplace+Wishlist/add-merchant-product-offer-from-wishlist-to-cart.gif) 

{% info_block warningBox "Developer guides" %}

> Are you a developer? See [Marketplace Wishlist](/docs/marketplace/dev/feature-walkthroughs/{{ page.version }}/marketplace-wishlist.html) feature walkthrough for developers.

Reviewer: ```suggestion
Are you a developer? See [Marketplace Wishlist feature walkthrough](/docs/marketplace/dev/feature-walkthroughs/{{ page.version }}/marketplace-wishlist.html) for developers.
```
